### ISSUE TITLE
recurring_select with shcedulable and ice_cube

### ISSUE THREAD

**Issue by Frank004** [benignware/schedulable#2]
Can we use recurring_select form helprs with shcedulable

**Comment by rexblack** [comment]
Sorry, this hasn't been tested yet. 
If you're using simple_form, then you can use the form-input that comes bundled with Schedulable. Simply run the generator: 
rails g schedulable:simple_form
You could also inspect the form-input's code and roll your own helpers. If you do so, you're welcome to contribute.
Before writing Schedulable, i checked out recurring_select and decided that it didn't fit my needs at that time. I assume that both modules do not share the same approach or data-model, but i'm not sure about this.

**Comment by Frank004** [comment]
YES I test it out and dnt work as need it im goint to try and build one where can I find the fileds that it use  to make one for rails forms helper
